.github/workflows/subgraph.yml

```
@@ -56,5 +56,9 @@ jobs:
       - name: Deploy subgraph
         run: npm run deploy-local -- --network local --version-label 0.0.1
 
+      - name: Sleep for 10 seconds
```

nit: so we know why the sleep in code
```suggestion
      - name: Sleep for 10 seconds to allow time to index post new sub-graph deployment
```

---

✅  Done